charts/rancher-logging/0.0.1/README.md

```diff
@@ -0,0 +1,49 @@
+# Rancher Logging
```

Comment:
Can `rancher-logging` use in another platform?

Reply:
what do you mean? We customize the fluentd for rancher